src/core/public/integrations/moment/moment_service.ts

```diff
@@ -35,7 +35,15 @@ export class MomentService implements CoreService {
   public async setup() {}
 
   public async start({ uiSettings }: StartDeps) {
-    const setDefaultTimezone = (tz: string) => moment.tz.setDefault(tz);
+    const setDefaultTimezone = (tz: string) => {
+      const timezones = moment.tz.names();
+      const hasTimezone = timezones.includes(tz);
```

Comment:
Looks like Moment has [moment.tz.zone](https://momentjs.com/timezone/docs/#/data-loading/checking-if-a-zone-exists/) which checks for existance, returning the zone if loaded or null otherwise.

Could something like this work?

`moment.tz.setDefault(moment.tz.zone(tz))`

Reply:
I gave it a try - zone ends up returning a `Zone` or null, and we have to access name.  It ends up fairly close to this, so I could go either way. lemme know

```
const zone = moment.tz.zone(tz)
if (zone) moment.tz.setDefault(zone.name)
```

Reply:
Ah, Zone is an object. I do think it's better to use `moment.tz.zone` to check for existence, though.